### PR TITLE
feat: add concurrenecy override

### PR DESCRIPTION
**Pull request by notrab**

### CODE REVIEW

**Review by ronakganatra — DISMISSED** [review]
🍰

File: gatsby-source-graphcms/README.md [hunk 1]
```diff
@@ -73,12 +73,13 @@ module.exports = {
...
 | `locales`             | String _(Default: `['en']`)_             | An array of locale key strings from your GraphCMS project. [Learn more](#querying-localised-nodes). You can read more about working with localisation in GraphCMS [here](https://graphcms.com/docs/guides/concepts/i18n).                                                                                                              |
 | `stages`              | String _(Default: `['PUBLISHED']`)_      | An array of Content Stages from your GraphCMS project. [Learn more](#querying-from-content-stages). You can read more about using Content Stages [here](https://graphcms.com/guides/working-with-content-stages).                                                                                                                      |
+| `queryConcurrency`    | Integer _(Default: 10)_                  | The number of promises ran at once when executing queries.                                                                                                                                                                                                                                                                             |
```
Comment (notrab) [inline gatsby-source-graphcms/README.md:82]: Might be worth just naming this `concurrency`

Comment (spences10) [reply gatsby-source-graphcms/README.md:82]: Agree

**Review by jpedroschmitz — APPROVED** [review]
LGTM

**Review by spences10 — APPROVED** [review]

**Pull request merged**